### ISSUE TITLE
Suggestion: More Dragon Controls

### ISSUE THREAD

**Issue by CTH999** [AlexModGuy/Ice-and-Fire-Suggestions#985]
So, for dragons, we can control them and make them breath fire and takle. That's cool and all, but I want to launch a fire bomb. I suggest that a keybind is added in to do just that!

**Comment by Hero-Artist13** [comment]
you already do that while doing fire breath
like it happens after a few seconds or a minute

**Comment by CTH999** [comment]
Cool! Granted, a specified key so we can do it immediately, and a bit more controll over it.

**Comment by chcknlvr** [comment]
YES great idea there is a bunch of attacks that wild dragons can do but tamed ones can't. They should definitely add this.
